source/developer/dev-setup-ubuntu-1604.rst

```diff
@@ -39,9 +39,13 @@ Set up your development environment for building, running, and testing Mattermos
 
        ``sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz``
 
-    c. Modify permissions on ``/usr/local/go``. Replace *{user}* and *{group}* with the user and group that you are logged in with.
+    c. Ensure the binary file of golang have the right permission, by running:
```

Comment:
Looks like the permissions are already 755 after extracting go from the tarball.

On my local Ubuntu install, I changed the owner:group of those files to root:root and the build works fine.

So maybe Step c is completely unnecessary and can be eliminated rather than changing it and adding Step d.